_rules/SC1-2-Video-description-track.md

```diff
@@ -17,18 +12,19 @@ test_aspects:
 authors:
 - Wilco Fiers
 - Brian Bors
+---
 
 ## Test Procedure
 
 ### Applicability
 
-The rule applies to any [non-streaming][] video element [visible in the page][] where the video contains [audio][] and a `track` element with a `kind="description"` attribute.
+The rule applies to any [non-streaming](#non-streaming) `video` element [visible on the page](#visible-on-the-page) where the video contains audio and a `track` element with a `kind="description"` attribute.
 
 ### Expectation
 
-The visual information of each test target not available through its audio is describes with a description track element.
+The visual information of each test target not available through its audio is described with a description `track` element.
```

Comment:
Check if this is more understandable
The visual information of each test target not available through its audio is described in the external resource specified in `track` element.

Reply:
I don't think this improves the readability of the sentence. I certainly had to read it multiple times to understand what you were suggesting.